### DISCUSSION TITLE
Guided Conversations in Python vs. Business Process Execution

### DISCUSSION THREAD

**Discussion by radrad** [microsoft/semantic-kernel#9140] [General]
I am in the process of understanding recent:
Guided Conversations in Python (https://github.com/microsoft/semantic-kernel/commit/ce6ccd7e86be3a8130f5cf9724fa5e22fe17c3f1)
and I am trying to convert the sample to c#. Any idea if this is planned?

Second, Is this overlapping with Business Process Execution with Semantic Kernel
https://github.com/microsoft/semantic-kernel/blob/main/docs/decisions/0054-processes.md

**Comment by evchaki** [comment]
@radrad - thanks for the ask here.  Process Framework will be the path for this moving forward.  We are taking the concepts of Guided Conversations and we will be showing how to use this in the Process Framework.  The Process Framework for Python will be coming in the next month or two.  Process Framework is available for C# - check out this blog post - https://devblogs.microsoft.com/semantic-kernel/integrating-ai-into-business-processes-with-the-process-framework/
